### PR TITLE
Hassio: select audio input/output for add-ons

### PR DESCRIPTION
**Pull request by c727**
audio

@pvizeli
api doesn't accept the keys atm: 
```
extra keys not allowed @ data['audio_input']. Got '0,0' extra keys not allowed @ data['audio_output']. Got None
```

### CODE REVIEW

File: hassio/addon-view/hassio-addon-audio.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/paper-card/paper-card.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+<link rel="import" href="../../bower_components/paper-dropdown-menu/paper-dropdown-menu.html">
+<link rel='import' href='../../bower_components/paper-listbox/paper-listbox.html'>
+<link rel='import' href='../../bower_components/paper-item/paper-item.html'>
+<link rel='import' href='../../bower_components/neon-animation/web-animations.html'>
```
Comment (balloob) [inline hassio/addon-view/hassio-addon-audio.html]: You're not using this?

**Review by balloob — APPROVED** [review]
Seriously stoked about this functionality! 🎉

**Comment by balloob** [comment]
Ok to merge when either import removed / my comment ignored because I missed where it is used.

File: hassio/addon-view/hassio-addon-audio.html (hunk 1 above)
Comment (c727) [reply hassio/addon-view/hassio-addon-audio.html]: web-animations? I need the for the paper-listbox

**Comment by c727** [comment]
I will merge after a test with the new supervisor :)

File: hassio/addon-view/hassio-addon-audio.html (hunk 1 above)
Comment (NovapaX) [reply hassio/addon-view/hassio-addon-audio.html]: Doesn't paper-dropdown-menu import that via paper-menu-button?
I don't see a direct use in this component, only secondary.

Comment (c727) [reply hassio/addon-view/hassio-addon-audio.html]: without you get this error and the dropdown doesn't close
```
Uncaught ReferenceError: KeyframeEffect is not defined
    at HTMLElement.configure (fade-in-animation.html:39)
    at HTMLElement._configureAnimations (neon-animation-runner-behavior.html:44)
    at HTMLElement.playAnimation (neon-animation-runner-behavior.html:114)
    at HTMLElement._renderOpened (iron-dropdown.html:214)
    at HTMLElement.__openedChanged (iron-overlay-behavior.html:597)
    at HTMLElement.nextAnimationFrame (iron-overlay-behavior.html:620)
```

**Pull request merged**